index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="style.css">
+
+
+    <title>Jean Michel Basquiat </title>
+    
+    <!--
+    <style type="text/css">
+      body {
+        font-size: 20px;
+      }
+      p.main {
+        color: black;
+      }
+      .main {
+        color: black;
+        font-weight: bold;
+
+      } 
+      -->
+      
+    </style>
+  </head>
+  <body>
+
+    <h1>Jean Michel Basquiat</h1>
+    
+    <nav>
```

Comment:
@elysemclark excellent indentation here :) Use this as an example for experience.html.